doc/stability_wasm_proposals.md

```diff
@@ -0,0 +1,74 @@
+# Wasm Proposals
+
+This document is intended to describe the current status of WebAssembly proposals and WASI proposals in WAMR.
+
+Only track proposals that are followed in the [WebAssembly proposals](https://github.com/WebAssembly/proposals) and [WASI proposals](https://github.com/WebAssembly/WASI/blob/main/Proposals.md).
+
+Normally, the document tracks proposals that are in phase 4. However, if a proposal in an earlier phase receives support, it will be added to the list below.
+
+The _status_ represents the configuration _product-mini/platforms/linux/CMakeLists.txt_. There may be minor differences between the top-level CMakeLists and platform-specific CMakeLists.
+
+Users can turn those features on or off by using compilation options. If a relevant compilation option is not available(`N/A`), it indicates that the feature is permanently enabled.
+
+## On-by-default Wasm Proposals
+
+| Proposal                              | Phase 4 | Compilation Option         |
```

Comment:
Marking some proposals as `Phase 4` is not verly accurate, in fact, `Bulk memory operations`, `Fixed-width SIMD`, `Multi-value`, `Non-trapping float-to-int conversions` and `Sign-extension operators` are finished proposals, and `Extended Constant Expressions` is in `Phase 5`, and some are in `Phase 4`. How about changing this column to `Phase` and setting its value to `Finished`, `Phase 5`, `Phase 4` or others?

Reply:
I intended to convey **pass phase 4**, but evidently, was not clear. Would *>= Phase 4* be more accurate?

Reply:
Yes, changing to `>= Phase 4` sounds better

Reply:
Or how about `Phase 4+`?